From c37cbd3d81762d131a2d43f827c22ca06344aef1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:35 -0500
Subject: [PATCH 009/407] B::C patch for comp/require.t

---
 t/CORE/comp/require.t | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/t/CORE/comp/require.t b/t/CORE/comp/require.t
index b3e4995..545b35b 100644
--- a/t/CORE/comp/require.t
+++ b/t/CORE/comp/require.t
@@ -1,10 +1,7 @@
 #!./perl
 
-BEGIN {
-    chdir 't' if -d 't';
-    @INC = '.';
-    push @INC, '../lib', '../ext/re';
-}
+chdir 't/CORE';
+unshift @INC, '.', 't/CORE/lib';
 
 sub do_require {
     %INC = ();
-- 
2.3.8 (Apple Git-58)

