From 704d83f1050bb7468949b8e8be30bd1c798b4d9d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:37 -0500
Subject: [PATCH 018/407] B::C patch for io/defout.t

---
 t/CORE/io/defout.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/defout.t b/t/CORE/io/defout.t
index a70fb6f..27123bc 100644
--- a/t/CORE/io/defout.t
+++ b/t/CORE/io/defout.t
@@ -7,9 +7,8 @@
 # it probably needs expanding at some point to cover other stuff.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
 }
 
 plan tests => 16;
-- 
2.3.8 (Apple Git-58)

