From e770d3e02eb2cc9bba6ce03f0ac96f624c6b2b6a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:37 -0500
Subject: [PATCH 019/407] B::C patch for io/dup.t

---
 t/CORE/io/dup.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/dup.t b/t/CORE/io/dup.t
index 1328aeb..62e057c 100644
--- a/t/CORE/io/dup.t
+++ b/t/CORE/io/dup.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require "./test.pl";
+    push @INC, qw{t/CORE/lib};
+    require 'test.pl';
 }
 
 use Config;
-- 
2.3.8 (Apple Git-58)

