From 9bc338ec3b041c91f45bacb197d80a9d9dcfa281 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:39 -0500
Subject: [PATCH 035/407] B::C patch for io/perlio_leaks.t

---
 t/CORE/io/perlio_leaks.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/perlio_leaks.t b/t/CORE/io/perlio_leaks.t
index a7cdf28..a51a6ba 100644
--- a/t/CORE/io/perlio_leaks.t
+++ b/t/CORE/io/perlio_leaks.t
@@ -2,9 +2,8 @@
 # ioleaks.t
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
 }
 
 use strict;
-- 
2.3.8 (Apple Git-58)

