From 92d163d0fae39f11faf4d1943b6059693dfd366b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:41 -0500
Subject: [PATCH 054/389] B::C patch for mro/basic_01_c3.t

---
 t/CORE/mro/basic_01_c3.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_01_c3.t b/t/CORE/mro/basic_01_c3.t
index bf32dd3..e6ce8c1 100644
--- a/t/CORE/mro/basic_01_c3.t
+++ b/t/CORE/mro/basic_01_c3.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 4);
+require 't/CORE/test.pl'; plan(tests => 4);
 
 =pod
 
-- 
2.3.8 (Apple Git-58)

