From 7f1fc904a13d64431ffe990bdb14b74c73a69b92 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:43 -0500
Subject: [PATCH 085/389] B::C patch for mro/inconsistent_c3.t

---
 t/CORE/mro/inconsistent_c3.t | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/t/CORE/mro/inconsistent_c3.t b/t/CORE/mro/inconsistent_c3.t
index b005226..1889827 100644
--- a/t/CORE/mro/inconsistent_c3.t
+++ b/t/CORE/mro/inconsistent_c3.t
@@ -1,16 +1,14 @@
 #!./perl
 
 BEGIN {
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
+    push @INC, "t/CORE/lib";
+    require q(test.pl);
 }
 
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 1);
+plan(tests => 1);
 
 require mro;
 
-- 
2.3.8 (Apple Git-58)

