From 1ff2141f3eff2ca0e3ffd01267ef5d8374e8bae7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:47 -0500
Subject: [PATCH 127/407] B::C patch for op/alarm.t

---
 t/CORE/op/alarm.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/alarm.t b/t/CORE/op/alarm.t
index 28cc65c..2fc8095 100644
--- a/t/CORE/op/alarm.t
+++ b/t/CORE/op/alarm.t
@@ -1,9 +1,7 @@
 #!./perl 
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 BEGIN {
-- 
2.3.8 (Apple Git-58)

