From a0a23871ea970a8c05d26aa663322a849ef2b9a1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 132/407] B::C patch for op/array_base.t

---
 t/CORE/op/array_base.t | 24 ++++++++++++++----------
 1 file changed, 14 insertions(+), 10 deletions(-)

diff --git a/t/CORE/op/array_base.t b/t/CORE/op/array_base.t
index a30236d..f6869cd 100644
--- a/t/CORE/op/array_base.t
+++ b/t/CORE/op/array_base.t
@@ -1,22 +1,26 @@
 #!perl -w
 use strict;
 
+my %begin_tests;
 BEGIN {
- chdir 't' if -d 't';
- require './test.pl';
+	use v5.15;
+    require 't/CORE/test.pl';
 
- plan (tests => my $tests = 11);
+    # Run these at BEGIN time, before arybase loads
+  	$begin_tests{'eval'} = eval('$[ = 1; 123');
+  	$begin_tests{'error'} = $@;
+}
+
+plan (tests => my $tests = 11);
 
- # Run these at BEGIN time, before arybase loads
- use v5.15;
- is(eval('$[ = 1; 123'), undef);
- like($@, qr/\AAssigning non-zero to \$\[ is no longer possible/);
+# delay test for B::C
+is($begin_tests{'eval'}, undef);
+like($begin_tests{'error'}, qr/\AAssigning non-zero to \$\[ is no longer possible/);
 
- if (is_miniperl()) {
-   # skip the rest
+if (is_miniperl()) {
+	# skip the rest
    SKIP: { skip ("no arybase.xs on miniperl", $tests-2) }
    exit;
- }
 }
 
 no warnings 'deprecated';
-- 
2.3.8 (Apple Git-58)

