From f5e236b67530e98f6e98971de5ab6ba6f8780cbd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 136/389] B::C patch for op/attrs.t

---
 t/CORE/op/attrs.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/attrs.t b/t/CORE/op/attrs.t
index 16e1fce..9599662 100644
--- a/t/CORE/op/attrs.t
+++ b/t/CORE/op/attrs.t
@@ -3,9 +3,8 @@
 # Regression tests for attributes.pm and the C< : attrs> syntax.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("miniperl can't load attributes");
 }
 
-- 
2.3.8 (Apple Git-58)

