From 460c1e021940f2e83a20e95eb42962d5057848e1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 150/389] B::C patch for op/context.t

---
 t/CORE/op/context.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/context.t b/t/CORE/op/context.t
index 41ee84b..155eb70 100644
--- a/t/CORE/op/context.t
+++ b/t/CORE/op/context.t
@@ -1,11 +1,9 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    require 't/CORE/test.pl';
 }
 
-require "./test.pl";
 plan( tests => 8 );
 
 sub foo {
-- 
2.3.8 (Apple Git-58)

