From 5a04406c112d211803f0a25b6a5ccc542d1c5ca8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 165/389] B::C patch for op/do.t

---
 t/CORE/op/do.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/do.t b/t/CORE/op/do.t
index 49c0de3..544fd4e 100644
--- a/t/CORE/op/do.t
+++ b/t/CORE/op/do.t
@@ -1,10 +1,9 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
+
 use strict;
 no warnings 'void';
 
-- 
2.3.8 (Apple Git-58)

