From 09c2202f05f9a48ad51a7dc57228a186bd3add84 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 173/389] B::C patch for op/exp.t

---
 t/CORE/op/exp.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/exp.t b/t/CORE/op/exp.t
index eb53f1b..376d676 100644
--- a/t/CORE/op/exp.t
+++ b/t/CORE/op/exp.t
@@ -3,9 +3,7 @@
 # Simple tests for the basic math functions.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 use Config;
-- 
2.3.8 (Apple Git-58)

