From abbad8ef1a2fd3891538bd2e8186aabf6ce0b431 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 188/407] B::C patch for op/grep.t

---
 t/CORE/op/grep.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/grep.t b/t/CORE/op/grep.t
index ea434a5..6cf90e9 100644
--- a/t/CORE/op/grep.t
+++ b/t/CORE/op/grep.t
@@ -5,9 +5,7 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require "./test.pl";
+    require "t/CORE/test.pl";
 }
 
 plan( tests => 66 );
-- 
2.3.8 (Apple Git-58)

