From 18a524e491f7bd6acbc0ce2eb6b080935ad505b6 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 190/389] B::C patch for op/gv.t

---
 t/CORE/op/gv.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/gv.t b/t/CORE/op/gv.t
index e695923..6efcbd4 100644
--- a/t/CORE/op/gv.t
+++ b/t/CORE/op/gv.t
@@ -5,13 +5,12 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    set_up_inc('../lib');
+    require 't/CORE/test.pl';
 }
 
 use warnings;
 
+chdir 't/CORE';
 plan( tests => 273 );
 
 # type coercion on assignment
-- 
2.3.8 (Apple Git-58)

