From c752c16675f9e2b847c65a210a536d0ef4a35ddc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 191/389] B::C patch for op/hash-rt85026.t

---
 t/CORE/op/hash-rt85026.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/hash-rt85026.t b/t/CORE/op/hash-rt85026.t
index 8dbf004..63f04f6 100644
--- a/t/CORE/op/hash-rt85026.t
+++ b/t/CORE/op/hash-rt85026.t
@@ -1,9 +1,7 @@
 #!/usr/bin/perl -w
 
 BEGIN {
-  chdir 't' if -d 't';
-  @INC = '../lib';
-  require './test.pl';
+  require "t/CORE/test.pl";
   skip_all_without_dynamic_extension("Devel::Peek");
 }
 
-- 
2.3.8 (Apple Git-58)

