From 35f10a5e6b5e31be758cc089d0180bef29f5738b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 193/407] B::C patch for op/hashassign.t

---
 t/CORE/op/hashassign.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/hashassign.t b/t/CORE/op/hashassign.t
index 57a625c..2eb103d 100644
--- a/t/CORE/op/hashassign.t
+++ b/t/CORE/op/hashassign.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 # use strict;
-- 
2.3.8 (Apple Git-58)

