From f6659cb7e5ba20581e702780efa16050bf834d9e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 199/407] B::C patch for op/incfilter.t

---
 t/CORE/op/incfilter.t | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/t/CORE/op/incfilter.t b/t/CORE/op/incfilter.t
index 9019ba5..a913906 100644
--- a/t/CORE/op/incfilter.t
+++ b/t/CORE/op/incfilter.t
@@ -3,16 +3,15 @@
 # Tests for the source filters in coderef-in-@INC
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require './test.pl';
-    skip_all_if_miniperl('no dynamic loading on miniperl, no Filter::Util::Call');
-    skip_all_without_perlio();
+    require "t/CORE/test.pl";
 }
 use strict;
 use Config;
 use Filter::Util::Call;
 
+skip_all_if_miniperl('no dynamic loading on miniperl, no Filter::Util::Call');
+skip_all_without_perlio();
+
 plan(tests => 153);
 
 unshift @INC, sub {
-- 
2.3.8 (Apple Git-58)

