From 1c05746e081d892327af11d86ba12863cef4cf20 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 205/389] B::C patch for op/kvhslice.t

---
 t/CORE/op/kvhslice.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/kvhslice.t b/t/CORE/op/kvhslice.t
index a5357ad..d3b98de 100644
--- a/t/CORE/op/kvhslice.t
+++ b/t/CORE/op/kvhslice.t
@@ -1,9 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    set_up_inc('../lib');
+    require "t/CORE/test.pl";
 }
 
 # use strict;
-- 
2.3.8 (Apple Git-58)

