From 24ac725033a83933d0b7b4c25f92b50ca65d19c8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 213/407] B::C patch for op/list.t

---
 t/CORE/op/list.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/list.t b/t/CORE/op/list.t
index c7056d0..1452060 100644
--- a/t/CORE/op/list.t
+++ b/t/CORE/op/list.t
@@ -1,9 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require "./test.pl";
+    require "t/CORE/test.pl";
 }
 
 plan( tests => 69 );
-- 
2.3.8 (Apple Git-58)

