From 32f2cafacb29c52df8291cfa1b9a569a3cf208dd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 215/389] B::C patch for op/localref.t

---
 t/CORE/op/localref.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/localref.t b/t/CORE/op/localref.t
index d481f15..3833d23 100644
--- a/t/CORE/op/localref.t
+++ b/t/CORE/op/localref.t
@@ -1,8 +1,6 @@
 #!./perl
 
-chdir 't' if -d 't';
-@INC = qw(. ../lib);
-require "./test.pl";
+require "t/CORE/test.pl";
 plan( tests => 64 );
 
 $aa = 1;
-- 
2.3.8 (Apple Git-58)

