From 8703d36ded260f3bd9369ec03d5493b3f16e82d3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 217/407] B::C patch for op/loopctl.t

---
 t/CORE/op/loopctl.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/loopctl.t b/t/CORE/op/loopctl.t
index e0e37e4..6ca8faf 100644
--- a/t/CORE/op/loopctl.t
+++ b/t/CORE/op/loopctl.t
@@ -31,9 +31,7 @@
 #
 #  -- .robin. <robin@kitsite.com>  2001-03-13
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require "./test.pl";
+  require "t/CORE/test.pl";
 }
 
 plan( tests => 67 );
-- 
2.3.8 (Apple Git-58)

