From 414c1a3782fccb961a8e35e17d784bb5892cc086 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 221/389] B::C patch for op/method.t

---
 t/CORE/op/method.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/method.t b/t/CORE/op/method.t
index 1171f4a..cf7d522 100644
--- a/t/CORE/op/method.t
+++ b/t/CORE/op/method.t
@@ -5,14 +5,14 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib lib ../dist/base/lib);
-    require "./test.pl";
+  unshift @INC, 'lib';
+  require "t/CORE/test.pl";
 }
 
 use strict;
 no warnings 'once';
 
+chdir 't/CORE';
 plan(tests => 147);
 
 @A::ISA = 'B';
-- 
2.3.8 (Apple Git-58)

