From 8ef0f8cb55cb01abf771428af77e8d83520cde94 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 222/407] B::C patch for op/mkdir.t

---
 t/CORE/op/mkdir.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/mkdir.t b/t/CORE/op/mkdir.t
index 2631526..1ab189d 100644
--- a/t/CORE/op/mkdir.t
+++ b/t/CORE/op/mkdir.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 22;
-- 
2.3.8 (Apple Git-58)

