From 952292fc092a0d04b1f4b17e5eec6a61444c342e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 235/407] B::C patch for op/packagev.t

---
 t/CORE/op/packagev.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/packagev.t b/t/CORE/op/packagev.t
index 86fb462..eab7a40 100644
--- a/t/CORE/op/packagev.t
+++ b/t/CORE/op/packagev.t
@@ -1,9 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw '../lib ../cpan/version/lib';
-    require './test.pl';
+    require "t/CORE/test.pl";
 }
 
 # XXX remove this later -- dagolden, 2010-01-13
-- 
2.3.8 (Apple Git-58)

