From 5434362863c78ead332e3f5d9ef502642617abb1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 236/407] B::C patch for op/pos.t

---
 t/CORE/op/pos.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/pos.t b/t/CORE/op/pos.t
index 04b5272..8de261e 100644
--- a/t/CORE/op/pos.t
+++ b/t/CORE/op/pos.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 29;
-- 
2.3.8 (Apple Git-58)

