From c91f66deaf9716e6376948f538cb9e8ca4e0c206 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 239/407] B::C patch for op/print.t

---
 t/CORE/op/print.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/print.t b/t/CORE/op/print.t
index 15f880f..54685a2 100644
--- a/t/CORE/op/print.t
+++ b/t/CORE/op/print.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    require "./test.pl";
+	require "t/CORE/test.pl";
 }
 
 plan(3);
-- 
2.3.8 (Apple Git-58)

