From bb7c601e8b22e0ce1988396b1b7715ca6644d5cd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 243/389] B::C patch for op/qr.t

---
 t/CORE/op/qr.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/qr.t b/t/CORE/op/qr.t
index 3ce7486..644d16c 100644
--- a/t/CORE/op/qr.t
+++ b/t/CORE/op/qr.t
@@ -3,8 +3,7 @@
 use strict;
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
+    require "t/CORE/test.pl";
 }
 
 plan(tests => 32);
-- 
2.3.8 (Apple Git-58)

