From 7ec143515a05aec8b853803d9c479674a8e7ec2d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 246/407] B::C patch for op/range.t

---
 t/CORE/op/range.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/range.t b/t/CORE/op/range.t
index 7809882..132efd7 100644
--- a/t/CORE/op/range.t
+++ b/t/CORE/op/range.t
@@ -1,11 +1,9 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib', '.');
+    require "t/CORE/test.pl";
 }   
 # Avoid using eq_array below as it uses .. internally.
-require './test.pl';
 
 use Config;
 
-- 
2.3.8 (Apple Git-58)

