From a957900fe2727c2302363b9353969e577a540ae0 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 251/407] B::C patch for op/ref.t

---
 t/CORE/op/ref.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/ref.t b/t/CORE/op/ref.t
index 19a44bb..1acc336 100644
--- a/t/CORE/op/ref.t
+++ b/t/CORE/op/ref.t
@@ -1,9 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require './test.pl';
+    require "t/CORE/test.pl";    
 }
 
 use strict qw(refs subs);
-- 
2.3.8 (Apple Git-58)

