From 861e6f5b1522f2cded67bc829e5bc453cf3f2f8c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 257/389] B::C patch for op/rt119311.t

---
 t/CORE/op/rt119311.t | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/t/CORE/op/rt119311.t b/t/CORE/op/rt119311.t
index dabc90e..851bdfd 100644
--- a/t/CORE/op/rt119311.t
+++ b/t/CORE/op/rt119311.t
@@ -22,7 +22,10 @@
 # outer sub.  Then we test a lexical directly inside the sub that DESTROY
 # calls.  Then we repeat with formats.
 
-BEGIN { chdir 't' if -d 't'; require './test.pl' }
+BEGIN {
+    require 't/CORE/test.pl';
+}
+
 plan 22;
 
 sub foo {
-- 
2.3.8 (Apple Git-58)

