From 0c49309638f679dd61b7d5b85df8720ddc7c0a4f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 267/389] B::C patch for op/sort.t

---
 t/CORE/op/sort.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/sort.t b/t/CORE/op/sort.t
index 4909474..1f0d5b8 100644
--- a/t/CORE/op/sort.t
+++ b/t/CORE/op/sort.t
@@ -1,9 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    set_up_inc('../lib');
+  require "t/CORE/test.pl";
 }
 use warnings;
 plan( tests => 183 );
-- 
2.3.8 (Apple Git-58)

