From 17b9c9b20644ff450631d7c0f5fd8caa735c1f71 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 268/389] B::C patch for op/splice.t

---
 t/CORE/op/splice.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/splice.t b/t/CORE/op/splice.t
index dde11f5..bb6b2df 100644
--- a/t/CORE/op/splice.t
+++ b/t/CORE/op/splice.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 $|  = 1;
-- 
2.3.8 (Apple Git-58)

