From 2ce43a88f3743005e6c517eb5a18d95d1bd4cd04 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 269/389] B::C patch for op/split.t

---
 t/CORE/op/split.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/split.t b/t/CORE/op/split.t
index 50579bf..e10993b 100644
--- a/t/CORE/op/split.t
+++ b/t/CORE/op/split.t
@@ -1,9 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    set_up_inc('../lib');
+    require "t/CORE/test.pl";
 }
 
 plan tests => 131;
-- 
2.3.8 (Apple Git-58)

