From 0b40debde29f64f73c2df41e49f636028ef745b4 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 270/407] B::C patch for op/split_unicode.t

---
 t/CORE/op/split_unicode.t | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/t/CORE/op/split_unicode.t b/t/CORE/op/split_unicode.t
index 8bfa935..b5f71bd 100644
--- a/t/CORE/op/split_unicode.t
+++ b/t/CORE/op/split_unicode.t
@@ -1,12 +1,12 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    skip_all_if_miniperl("no dynamic loading on miniperl, no File::Spec (used by charnames)");
-    plan(tests => 145);
+    require "t/CORE/test.pl";    
 }
 
+skip_all_if_miniperl("no dynamic loading on miniperl, no File::Spec (used by charnames)");    
+plan(tests => 145);
+
 {
     # check the special casing of split /\s/ and unicode
     use charnames qw(:full);
-- 
2.3.8 (Apple Git-58)

