From 28b3d614a911ff332680f01771bb7833244097a9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:00 -0500
Subject: [PATCH 278/407] B::C patch for op/study.t

---
 t/CORE/op/study.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/study.t b/t/CORE/op/study.t
index 906aba9..c055315 100644
--- a/t/CORE/op/study.t
+++ b/t/CORE/op/study.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 watchdog(10);
-- 
2.3.8 (Apple Git-58)

