From 958c6fdfae8ad69ab1c3a210d1edc5cd71c73c9f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:00 -0500
Subject: [PATCH 283/389] B::C patch for op/svleak.t

---
 t/CORE/op/svleak.t | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/t/CORE/op/svleak.t b/t/CORE/op/svleak.t
index 076f2bf..3c3fd45 100644
--- a/t/CORE/op/svleak.t
+++ b/t/CORE/op/svleak.t
@@ -5,10 +5,7 @@
 # see if the count increases.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
-
+    require "t/CORE/test.pl";
     eval { require XS::APItest; XS::APItest->import('sv_count'); 1 }
 	or skip_all("XS::APItest not available");
 }
-- 
2.3.8 (Apple Git-58)

