From 367986df9666327044727dfbc6f47bd0ca9bed99 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:01 -0500
Subject: [PATCH 294/407] B::C patch for op/undef.t

---
 t/CORE/op/undef.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/undef.t b/t/CORE/op/undef.t
index ddef596..bd6c8e0 100644
--- a/t/CORE/op/undef.t
+++ b/t/CORE/op/undef.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.3.8 (Apple Git-58)

