From 5ef0e61a8a9ae209d2143a9021d035cb8c1d3874 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:01 -0500
Subject: [PATCH 296/407] B::C patch for op/unlink.t

---
 t/CORE/op/unlink.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/unlink.t b/t/CORE/op/unlink.t
index 892b646..3e67419 100644
--- a/t/CORE/op/unlink.t
+++ b/t/CORE/op/unlink.t
@@ -1,9 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+	require "t/CORE/test.pl";
 }
 
 plan 6;
-- 
2.3.8 (Apple Git-58)

