From 4e37f8b614010807f9a81b3b77bbb3f88090bb86 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 297/407] B::C patch for op/unshift.t

---
 t/CORE/op/unshift.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/unshift.t b/t/CORE/op/unshift.t
index 7782b2a..4434439 100644
--- a/t/CORE/op/unshift.t
+++ b/t/CORE/op/unshift.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    require "./test.pl";
+    require "t/CORE/test.pl";
 }
 
 plan(36);
-- 
2.3.8 (Apple Git-58)

