From 78cf7ecc3dd22a3c5ee7d65e84d222a9565a4d5f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 299/407] B::C patch for op/utf8cache.t

---
 t/CORE/op/utf8cache.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/utf8cache.t b/t/CORE/op/utf8cache.t
index e897587..6b0d7d2 100644
--- a/t/CORE/op/utf8cache.t
+++ b/t/CORE/op/utf8cache.t
@@ -2,9 +2,8 @@
 # Test for malfunctions of utf8 cache
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.3.8 (Apple Git-58)

