From a1f5e82a9050decc1ad6a0435aac4086de0c6dd8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 301/389] B::C patch for op/utf8magic.t

---
 t/CORE/op/utf8magic.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/utf8magic.t b/t/CORE/op/utf8magic.t
index c071664..37db084 100644
--- a/t/CORE/op/utf8magic.t
+++ b/t/CORE/op/utf8magic.t
@@ -1,9 +1,8 @@
 #!perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl'; require './charset_tools.pl';
+	require "t/CORE/test.pl";
+	require "t/CORE/charset_tools.pl";
 }
 
 plan tests => 6;
-- 
2.3.8 (Apple Git-58)

