From feb40d962be50090edcdf54c482ab11f56032044 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 308/389] B::C patch for op/warn.t

---
 t/CORE/op/warn.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/warn.t b/t/CORE/op/warn.t
index 42b88f8..44565bb 100644
--- a/t/CORE/op/warn.t
+++ b/t/CORE/op/warn.t
@@ -2,9 +2,9 @@
 #line 3 warn.t
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl'; require './charset_tools.pl';
-    set_up_inc('../lib');
+    require "t/CORE/test.pl";
+    require "t/CORE/charset_tools.pl";
+    # Keep file len the same  so warning messages come out right.
 }
 
 plan 32;
-- 
2.3.8 (Apple Git-58)

