From de3c792662d33ec72962f333e7935fce8996e70b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 310/407] B::C patch for op/write.t

---
 t/CORE/op/write.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/op/write.t b/t/CORE/op/write.t
index 590d658..202a785 100644
--- a/t/CORE/op/write.t
+++ b/t/CORE/op/write.t
@@ -1,9 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    set_up_inc('../lib');
+    require "t/CORE/test.pl";
 }
 
 use strict;	# Amazed that this hackery can be made strict ...
-- 
2.3.8 (Apple Git-58)

