From 24e15103a00a5e53d2c3f9f8399473807b219d4a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 321/389] B::C patch for re/pat_special_cc.t

---
 t/CORE/re/pat_special_cc.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/re/pat_special_cc.t b/t/CORE/re/pat_special_cc.t
index d116eb9..4537c8e 100644
--- a/t/CORE/re/pat_special_cc.t
+++ b/t/CORE/re/pat_special_cc.t
@@ -6,9 +6,7 @@
 # \w/\W.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib','.');
-    require './test.pl';
+    require "t/CORE/test.pl";
 }
 
 use strict;
-- 
2.3.8 (Apple Git-58)

