From 8c772cefc74b398de662bc882a740cc6999256c2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:06 -0500
Subject: [PATCH 356/407] B::C patch for uni/cache.t

---
 t/CORE/uni/cache.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/uni/cache.t b/t/CORE/uni/cache.t
index 4cd9a48..d4ce738 100644
--- a/t/CORE/uni/cache.t
+++ b/t/CORE/uni/cache.t
@@ -1,7 +1,5 @@
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    set_up_inc('../lib');
+	require "t/CORE/test.pl";
     skip_all_without_unicode_tables();
 }
 
-- 
2.3.8 (Apple Git-58)

