From f1fed40547b545af0cdd8de819eeb67c77a732b7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 359/389] B::C patch for uni/chomp.t

---
 t/CORE/uni/chomp.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/uni/chomp.t b/t/CORE/uni/chomp.t
index ea86a25..fe3576c 100644
--- a/t/CORE/uni/chomp.t
+++ b/t/CORE/uni/chomp.t
@@ -1,8 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
+	require "t/CORE/test.pl";
     skip_all_without_dynamic_extension('Encode');
     skip_all("encoding doesn't work with EBCDIC") if $::IS_EBCDIC;
     skip_all_without_perlio();
-- 
2.3.8 (Apple Git-58)

