From 173ec2626116b546a3780dd78f1d06087c82e1b5 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 361/407] B::C patch for uni/class.t

---
 t/CORE/uni/class.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/uni/class.t b/t/CORE/uni/class.t
index 37392aa..361751d 100644
--- a/t/CORE/uni/class.t
+++ b/t/CORE/uni/class.t
@@ -1,7 +1,5 @@
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    set_up_inc(qw(../lib .));
+    require "t/CORE/test.pl";
     skip_all_without_unicode_tables();
 }
 
-- 
2.3.8 (Apple Git-58)

