From 7545e6f187eed374c4dff177a21fde90fe29d3d3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 368/389] B::C patch for uni/latin2.t

---
 t/CORE/uni/latin2.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/uni/latin2.t b/t/CORE/uni/latin2.t
index ba67e09..a4d0998 100644
--- a/t/CORE/uni/latin2.t
+++ b/t/CORE/uni/latin2.t
@@ -1,8 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
+    require "t/CORE/test.pl";
     skip_all_without_dynamic_extension('Encode');
     skip_all("no encoding pragma in EBCDIC") if $::IS_EBCDIC;
     skip_all_without_perlio();
-- 
2.3.8 (Apple Git-58)

