From 5fcf32726e548e175c705244dfc6c3c9b20ac4ab Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 370/389] B::C patch for uni/lower.t

---
 t/CORE/uni/lower.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/uni/lower.t b/t/CORE/uni/lower.t
index 1301b48..c834886 100644
--- a/t/CORE/uni/lower.t
+++ b/t/CORE/uni/lower.t
@@ -1,6 +1,5 @@
 BEGIN {
-    chdir 't' if -d 't';
-    require "uni/case.pl";
+    require "t/CORE/uni/case.pl";
 }
 
 use feature 'unicode_strings';
-- 
2.3.8 (Apple Git-58)

