From 1bac686558b5916c47b6e471638f9ab468618f85 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 375/407] B::C patch for uni/parser.t

---
 t/CORE/uni/parser.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/uni/parser.t b/t/CORE/uni/parser.t
index ad905b0..0e78dcf 100644
--- a/t/CORE/uni/parser.t
+++ b/t/CORE/uni/parser.t
@@ -4,8 +4,7 @@
 # (including weird syntax errors)
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
+    require "t/CORE/test.pl";
     skip_all_without_unicode_tables();
 }
 
-- 
2.3.8 (Apple Git-58)

