From e419e1647923f44155455d5262fc6844220f93c7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 378/389] B::C patch for uni/sprintf.t

---
 t/CORE/uni/sprintf.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/uni/sprintf.t b/t/CORE/uni/sprintf.t
index 258ab54..35fceff 100644
--- a/t/CORE/uni/sprintf.t
+++ b/t/CORE/uni/sprintf.t
@@ -1,9 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib .);
-    require "./test.pl";
+    require "t/CORE/test.pl";
 }
 
 plan tests => 52;
-- 
2.3.8 (Apple Git-58)

