From 1ec6c9abf8599d7f05b3c0cb40a2fe921ee121d3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 379/407] B::C patch for uni/stash.t

---
 t/CORE/uni/stash.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/uni/stash.t b/t/CORE/uni/stash.t
index 31d6c9d..5b033a1 100644
--- a/t/CORE/uni/stash.t
+++ b/t/CORE/uni/stash.t
@@ -5,9 +5,7 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    set_up_inc('../lib');
+    require "t/CORE/test.pl";
 }
 
 use utf8;
-- 
2.3.8 (Apple Git-58)

