From 7ea7158a6a5c2bb2c06084f5c3ceddff1bc6a059 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 380/389] B::C patch for uni/tie.t

---
 t/CORE/uni/tie.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/uni/tie.t b/t/CORE/uni/tie.t
index c476306..f78f612 100644
--- a/t/CORE/uni/tie.t
+++ b/t/CORE/uni/tie.t
@@ -1,8 +1,7 @@
 #!perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
+    require "t/CORE/test.pl";
 }
 
 plan (tests => 10);
-- 
2.3.8 (Apple Git-58)

