From 8a3f5b292cfc6a7c7e0ef57370553294e12d3eea Mon Sep 17 00:00:00 2001
From: Todd Rinaldo <toddr@cpan.org>
Date: Wed, 19 Jul 2017 18:36:12 -0500
Subject: [PATCH 5/5] Provide a way to clear swash invlists for B::C

---
 SOURCES/perl/lib/utf8_heavy.pl | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/SOURCES/perl/lib/utf8_heavy.pl b/SOURCES/perl/lib/utf8_heavy.pl
index 66c968aa..b43f2e8d 100644
--- a/SOURCES/perl/lib/utf8_heavy.pl
+++ b/SOURCES/perl/lib/utf8_heavy.pl
@@ -44,6 +44,12 @@ sub _loose_name ($) {
     # progress, pushed upon entry, popped upon return.
     my @recursed;

+    sub reset_swash {
+        undef $min_floating_slop;
+        @recursed = ();
+        %Cache = ();
+    }
+
     sub SWASHNEW {
         my ($class, $type, $list, $minbits, $none) = @_;
         my $user_defined = 0;
--
2.13.2

