From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 30 Jun 2017 11:28:31 -0600
Subject: [PATCH 43/44] better experience with gdb

---
 SOURCES/perl/perl.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/SOURCES/perl/perl.h b/SOURCES/perl/perl.h
index f8f0069d..161f7147 100644
--- a/SOURCES/perl/perl.h
+++ b/SOURCES/perl/perl.h
@@ -352,6 +352,11 @@
 #  endif
 #endif

+/* better experience with gdb */
+#ifndef PERL_GCC_BRACE_GROUPS_FORBIDDEN
+#    define PERL_GCC_BRACE_GROUPS_FORBIDDEN
+#endif
+
 /* Use PERL_UNUSED_RESULT() to suppress the warnings about unused results
  * of function calls, e.g. PERL_UNUSED_RESULT(foo(a, b)).
  *
