From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 14 Aug 2017 11:02:32 -0600
Subject: [PATCH 53/53] extra protection in Perl_sv_vcatpvfn_flags

---
 SOURCES/perl/sv.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/SOURCES/perl/sv.c b/SOURCES/perl/sv.c
index 03c91166..bb0d19d2 100644
--- a/SOURCES/perl/sv.c
+++ b/SOURCES/perl/sv.c
@@ -11554,9 +11554,11 @@ Perl_sv_vcatpvfn_flags(pTHX_ SV *const sv, const char *const pat, const STRLEN p
 			has_precis = TRUE;
 		    }
 		    argsv = MUTABLE_SV(va_arg(*args, void*));
-		    eptr = SvPV_const(argsv, elen);
+		    if (argsv) {
+            eptr = SvPV_const(argsv, elen);
 		    if (DO_UTF8(argsv))
 			is_utf8 = TRUE;
+            }
 		    goto string;
 		}
 		else if (n==2 || n==3) {	/* HEKf */
